src/Components/alert/Alert.css

```diff
@@ -0,0 +1,71 @@
+:root {
+  --color-1: #f44336;
+  --color-2: #ff9800;
+  --color-3: #2196f3;
+  --color-4: #4caf50;
+}
+
+.alert-container {
+  grid-area: pro;
+  padding: 0;
+  margin: 2rem;
+}
+
+.alert-flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  padding: 2rem;
+  border: 1px solid lightgray;
+  border-bottom-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  margin-bottom: 2rem;
+}
+
+.alert-flex-item {
+  background-color: var(--color-1);
+  border-bottom-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;

```

Comment:
Use shorthand property for border radius.

Reply:
Yeah, I will replace it with "border-radius". Thanks for the suggestion.